docs/docs/sample-scripts/entra/find-obsolete-m365-groups/index.mdx

```diff
@@ -185,37 +197,14 @@ Use this script to create a report of all M365 groups that are possibly obsolete
     param (
       [Parameter(Mandatory = $true)] [GroupInfo] $Group
     )
-
```

Comment:
delete date parsing due to introduction of new attribute `LastItemUserModifiedDate`instead of `LastContentModifiedDate`